back/spec/services/url_validation_service_spec.rb

```diff
@@ -69,7 +70,8 @@
       'https://facebook.com',
       'https://konveio.com/something',
       'https://subdomain.arcgis.com',
-      'https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pd'
+      'https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pd',
+      'https://btt.abalancingact\dummy'
```

Comment:
I would replace the `\` with a `/`, because you want to express that the missing `.com` part makes the URL invalid.